packages/@aws-cdk/aws-appmesh/lib/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './shared-interfaces';
 export * from './virtual-node';
 export * from './virtual-router';
 export * from './virtual-service';
+export * from './virtual-listener';
```

Comment:
I think we should rename this to `virtual-node-listener`. No reason to not be specific